src/EmailPage/Email.stories.js

```diff
@@ -5,10 +5,22 @@ import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 
 import Layout from '../Layout';
-import Email from './';
+import { CheckRecoveryLink, CheckMagicLink } from './';
 
-storiesOf('Email', module).add('Default', () => (
-  <Layout action={action}>
-    <Email email="example@kiwi.com" />
-  </Layout>
-));
+storiesOf('CheckEmail', module)
+  .add('Magical Link', () => (
+    <Layout action={action}>
+      <CheckMagicLink
+        email="example@kiwi.com"
+        text="To sign in, just click the link in the email we sent to"
+      />
+    </Layout>
+  ))
+  .add('Recovery Link', () => (
+    <Layout action={action}>
+      <CheckRecoveryLink
```

Comment:
I'm curious - it doesn't seem like `CheckRecoveryLink` and `CheckMagicLink` accept any props

Reply:
true, thanks.